test/test_util_error.cpp

```diff
@@ -95,6 +95,7 @@ TEST(BasicSystemErrors_Messages)
 #endif
     }
 
+#if !REALM_HAVE_CLANG_FEATURE(undefined_behavior_sanitizer)
```

Comment:
This test is verifying that we do something sensible given garbage input and UBSan correctly points out that the input is garbage.